packages/legend-query/src/stores/QueryBuilderSetupState.ts

```diff
@@ -35,6 +36,7 @@ import {
 export class QueryBuilderSetupState {
   queryBuilderState: QueryBuilderState;
   _class?: Class | undefined;
+  versionPropertyParameter?: ValueSpecification | undefined;
   mapping?: Mapping | undefined;
```

Comment:
`versionPropertyParameter` -> `_classVersionValue`. Open to suggestions for naming this :)